pkg/registries/registries.go

```diff
@@ -96,27 +110,46 @@ func mergedMirrorSets(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy
 }
 
 // EditRegistriesConfig edits, IN PLACE, the /etc/containers/registries.conf configuration provided in config, to:
-// - Mark whole registries in insecureRegistries as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
-// - Mark whole registries in blockedRegistries as blocked (any attempts to access them fail)
+// - Mark scope entries in insecureScopes as insecure (TLS is not required, and TLS certificate verification is not required when TLS is used)
+// - Mark scope entries in blockedScopes as blocked (any attempts to access them fail)
 // - Implement ImageContentSourcePolicy rules in icspRules.
-// “Whole registries” above means that the configuration applies to everything on that registry, including any possible separately-configured
+// "scopes" can be any of whole registries, which means that the configuration applies to everything on that registry, including any possible separately-configured
 // namespaces/repositories within that registry.
-func EditRegistriesConfig(config *sysregistriesv2.V2RegistriesConf, insecureRegistries, blockedRegistries []string, icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy) error {
+// or can be wildcard entries, which means that we accept wildcards in the form of *.example.registry.com for insecure and blocked registries only. We do not
```

Comment:
This still needs to document that registry/namespace/…[/repo] is a valid scope (or just refer to sysregistriesv2.Registry.Prefix ).